src/main/java/model/MountType.java

```diff
@@ -0,0 +1,15 @@
+package model;
+
+public enum MountType {
+    CEILING(1), WALL(2);
+
+    private int numberMountType;
+
+    MountType(int numberMountType) {
+    }
+
+    public static MountType getMountType(int number) {
+        MountType[] mountTypes = MountType.values();
```

Comment:
Let's implement a bit another logic. For each mountType compare method param with mountType.number. If equals return this mountType. If no one equals return null or throw exception